CMakeLists.txt

```diff
@@ -282,6 +282,7 @@ IF(WANT_SWH)
 		SET(LMMS_HAVE_SWH TRUE)
 		SET(STATUS_SWH "OK")
 	ELSE()
+		SET(LMMS_HAVE_SWH 0)
```

Comment:
This doesn't fix the problem mentioned in the previous review. Firstly, an undefined variable is considered falsy, so setting this to zero here won't change anything. Secondly, this variable isn't used, so no matter what you set it to, it won't have any effect. Instead, change the `if` commands in `plugins/LadspaEffect/CMakeLists.txt` to check `LMMS_HAVE_*`, not `WANT_*`.

Reply:
removed it.

Reply:
also, the conditions were changed in #6903